### PR TITLE
Add rest props to internal div

### PR DESCRIPTION
**Pull request by giulioz**
#### :boom: Breaking Changes

- Nothing

#### :rocket: Enhancements

- Needed if used in flex layouts where 100% size breaks the layout.

### CODE REVIEW

File: packages/vx-responsive/src/components/ParentSize.js [hunk 1]
```diff
@@ -43,9 +43,9 @@ export default class ParentSize extends React.Component {
     this.target = ref;
   }
   render() {
-    const { className, children } = this.props;
+    const { className, children, ...rest } = this.props;
```
Comment (hshoff) [inline packages/vx-responsive/src/components/ParentSize.js]: Super minor nit, but to match the rest of the codebase could we rename this `...restProps`?

Comment (giulioz) [reply packages/vx-responsive/src/components/ParentSize.js]: Done, sorry, we were in a rush at work and we missed this.
Thank you

**Comment by hshoff** [comment]
LGTM, thanks for the contribution!

**Comment by hshoff** [comment]
This is now published in `@vx/responsive@0.0.177`

**Pull request merged**